test/functional/services/remote/leadfoot_command.js

```diff
@@ -8,18 +8,25 @@ const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 
 let attemptCounter = 0;
-async function attemptToCreateCommand(log, server, chromedriverApi) {
+async function attemptToCreateCommand(log, server, driverApi) {
   const attemptId = ++attemptCounter;
+  let capabilities = {};
+
+  switch(driverApi.getBrowserName()) {
+    default:
+      capabilities = { browserName: 'firefox' };
```

Comment:
This looks like it's always going to launch firefox.

Reply:
Fixed.